client/my-sites/customer-home/cards/tasks/connect-accounts.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import QueryPublicizeConnections from 'components/data/query-publicize-connections';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ConnectAccountsTask = ( { skipTask, siteId, siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<ActionPanel className="tasks connect-accounts">
+			<QueryPublicizeConnections selectedSite />
+			<ActionPanelBody>
+				{ ! isMobile() && (
+					<ActionPanelFigure align="right">
+						<img src="/calypso/images/stats/tasks/social-links.svg" alt="" />
+					</ActionPanelFigure>
+				) }
+				<div className="tasks__timing">
+					<Gridicon icon="time" size={ 18 } />
+					<p>{ translate( '3 minutes' ) }</p>
+				</div>
+				<ActionPanelTitle>{ translate( 'Drive traffic to your site' ) }</ActionPanelTitle>
+				<p className="tasks__description">
+					{ translate(
+						'Integrate your site with social media to automatically post your content and drive traffic to your site.'
+					) }
+				</p>
+				<ActionPanelCta>
+					<Button
+						className="tasks__action"
+						primary
+						href={
+							isMobile()
```

Comment:
Do we need a different URL since the guided tour already specifies `when: isDesktop`?

Reply:
I think we still do. 

<img width="378" alt="Screenshot 2020-04-13 at 12 03 20" src="https://user-images.githubusercontent.com/43215253/79115816-c7396480-7d7e-11ea-8700-bb8e294efdb2.png">

My understanding, although I may be wrong here since this is just based on checking out the few examples in Calypso, is that `?tour=marketingConnectionsTour` forces the Guided Tour to appear. I'm not exactly sure the other ways in which it's done, but I reckon this is ideal for this case because otherwise I think you'd run into the issue mentioned in #41038, where people could dismiss the Guided Tour but end up not connecting their social media accounts under the assumption that they can just do it later, or skip it if they don't want to.